src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,141 @@
+:root {
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray-color: #616070;
+  --dark-blue-color: #060b35;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
```

Comment:
Consider using paddings here. For making nested elements follow the full width of the container